go-runtime/compile/parser.go

```diff
@@ -130,12 +131,35 @@ func (d *directiveCronJob) String() string {
 	return fmt.Sprintf("cron %s", d.Cron)
 }
 
+type directiveRetry struct {
+	Pos schema.Position
+
+	Count      *int   `parser:"'retry' (@Number Whitespace)?"`
+	MinBackoff string `parser:"@(Number Ident)"`
```

Comment:
I think you'll need something like `@(Number (?! Whitespace) Ident)` here to catch cases like `1 m`

Reply:
Done, and added a test for it 👍